src/dbnode/persist/fs/commitlog/commit_log.go

```diff
@@ -45,17 +45,25 @@ var (
 	timeZero = time.Time{}
 )
 
+// WritesBatch is a batch of Write.
+type WritesBatch []Write
```

Comment:
Can we wrap this up in a struct (which we could then pool)?

Reply:
Can we also rename it to `WriteBatch` and drop the `s`?  Just since in other places where we have write batches that's what we call them.